### ISSUE TITLE
What is the RD Program?

### ISSUE THREAD

**Issue by bniaulin** [msrd/Summit-2019-Unconference#33]
On the RD site, it's noted that the MVP Program is "deep technical focused" whereas the RD program is cross-technology (beyond Microsoft), customer experience focused and business focused.
However, the DL and the Summit is often used for "features" or "issues" that I would have assumed would be directed more towards the Product Teams at the engineering level. Aka MVP program.

And leverage the RD program to bring things like RDPerspectives, what's happening in specific industries, strategy/vision and ideas for new programs that we can help lift off. Bring in feedback at a more business level.
To provide help and guidance, and in return be shared vision/strategy/KPIs and 3 year roadmaps.

Perhaps we could start by understanding together what the RD Program is and what it isn't?
Or perhaps it's just me :)

**Comment by markusegger** [comment]
Probably can be combined with #11

**Comment by bniaulin** [comment]
I would disagree, #11 talks about the visibility of the program to get more awareness out there.
One could follow the other though absolutely.

**Comment by markusegger** [comment]
Yeah, but if I rank this on its own, it is going to completely drop off the list because it doesn't have enough votes. I think it would be better to have a reminder to discuss this in the same time slot, that is better than losing it.

In general, I do not mean these "merged sessions" as "duplicates we should drop", but instead, I want to make sure we do NOT drop them and get to discuss them at some level, at least. We do have pretty limited time, unfortunately.
